### PR TITLE
Update fileType cookbook example

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** Summary of changes

Addresses [CUMULUS-670]

Update docs with changes made in https://github.com/nasa/cumulus/pull/867

### CODE REVIEW

File: docs/data-cookbooks/setup.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ The schema for collections can be found [here](https://github.com/nasa/cumulus/t
 |:---:|:-----:|:--------:|-----------|
 |regex|`"^MOD09GQ\\.A[\\d]{7}\\.[\\S]{6}\\.006\\.[\\d]{13}\\.hdf$"`|Yes|Regex used to identify the file|
 |sampleFileName|`MOD09GQ.A2017025.h21v00.006.2017034065104.hdf"`|Yes|Filename used to validate the provided regex|
+|fileType|`"data"`|No|Value to be assigned to the Granule File Filetype.  CNM filetypes used by Cumulus CMR steps, non-CNM values will be treated as 'data' fileType|
```
Comment (markdboyd) [inline docs/data-cookbooks/setup.md]: A couple thoughts:

- Doesn't seem like the words `Granule File Filetype` should be capitalized?
- `CNM filetypes used by Cumulus CMR steps` should be `CNM filetypes will be used by Cumulus as fileType for CMR steps`?

File: packages/api/models/schemas.js [hunk 2]
```diff
@@ -145,7 +145,7 @@ module.exports.collection = {
           },
           fileType: {
             title: 'File Type',
-            description: 'CNM Filetype.  Used for submission to CMR.  Non CNM Filetypes default to "data"',
+            description: 'CNM Filetype.  Cumulus uses this for CMR submission.  Non-CNM filetypes will be treated as "data" fileType',
```
Comment (markdboyd) [inline packages/api/models/schemas.js]:
```suggestion
            description: 'CNM file type.  Cumulus uses this for CMR submission.  Non-CNM file types will be treated as "data" fileType',
```

I may be totally wrong, but do we only want to use the compound variable name  `fileType` when we're explicitly referring to the variable?

File: docs/data-cookbooks/setup.md (hunk 1 above)
Comment (markdboyd) [reply docs/data-cookbooks/setup.md]:
```suggestion
|fileType|`"data"`|No|Value to be assigned to the granule file fileType.  CNM file types will be used by Cumulus CMR steps as the fileType, non-CNM values will be treated as the 'data' fileType|
```

File: packages/api/models/schemas.js (hunk 2 above)
Comment (Jkovarik) [reply packages/api/models/schemas.js]: 👍  excellent point.

**Pull request merged**